rust/agama-cli/src/logs.rs

```diff
@@ -19,6 +19,9 @@ pub enum LogsCommands {
         #[clap(long, short = 'v')]
         /// Verbose output
         verbose: bool,
+        #[clap(long, short = 'd')]
+        /// Destination path
```

Comment:
Is this a file name or a directory?
If a directory, what is the default file name?